lib/super_digit.rb

```diff
@@ -1,11 +1,16 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n)
```

Comment:
What is n?  This will go a number of times depending on the number of digits or log<sub>10</sub>n times.